enhancements/installer/aws-lb-subnet-selection.md

```diff
@@ -0,0 +1,686 @@
+---
+title: aws-lb-subnet-selection
+authors:
+- "@gcs278"
+reviewers:
+- "@candita"
+- "@frobware"
+- "@rfredette"
+- "@alebedev87"
+- "@miheer"
+- "@Miciah"
+- "@mtulio"
+approvers:
+- "@patrickdillon"
+api-approvers:
+- None
+creation-date: 2024-05-29
+last-updated: 2024-06-20
+tracking-link:
+  - https://issues.redhat.com/browse/CORS-3440
+see-also:
+  - "enhancements/ingress/lb-subnet-selection-aws.md"
+replaces:
+superseded-by:
+---
+
+# Installer LoadBalancer Subnet Selection for AWS
+
+## Summary
+
+This enhancement extends the OpenShift Installer's install-config, enabling cluster admins to
+configure subnets for AWS load balancers created for their IngressControllers at install time.
+This configuration sets the subnets for both the default IngressController and the default
+subnets for all new IngressControllers at install time.
+
+This enhancement also deprecates the existing install-config field `platform.aws.subnets`
+in favor for a more flexible configuration field that handles both IngressController subnet
+selection (ingress subnets) and cluster resource subnet selection (cluster subnets).
```

Comment:
We don't have the latter as a use case.  Why is it being covered in this proposal?